introduction.md

```diff
@@ -1,3 +1,8 @@
 #introduction#
 ##My Git Dojo##
-I like Code School path Ruby and Git Dojo.
\ No newline at end of file
+I like Code School path Ruby and Git Dojo.
+
+*******************
+*Ruby
+*Java
+*JavaScript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?